Release 4.4.0 (#73)
blackbaud/skyux-phone-field: CHANGELOG.md

# 4.4.0 (2020-10-05)

> - Added support to the phone field component for specifiying if phone number extenstions are allowed. [#71](https://github.com/blackbaud/skyux-phone-field/pull/71)

Reviewer: ```suggestion
- Added support to the phone field component for specifying whether to allow phone number extensions. [#71](https://github.com/blackbaud/skyux-phone-field/pull/71)
```
